uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
FWIW, @EliAndrewC, she has a pretty high tolerance.  As an example, I'm still alive even though she has been able to drive herself to cons/fests/events for many years.